### PR TITLE
Career Folder

### PR DESCRIPTION
**Pull request by SaranshDawra**
Fixes: #1205 
@Kundan28 @vegetabill, please review the pull request.

### CODE REVIEW

**Comment by Kundan28** [comment]
Why you closed #1239 
This is not a good practice, you should learn how to fix conflicts instead of just deleting it and making a new PR.
I think you should open that PR and work on that itself instead of creating this new one.

**Comment by SaranshDawra** [comment]
@Kundan28 Sir please can you guide me how to resolve conflicts. As I am new to this.

**Comment by Kundan28** [comment]
@SaranshDawra For this type of issues you should better search the web and you will get the answer.
Go through the link below and you will understand everything.
https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-using-the-command-line

**Comment by Kundan28** [comment]
> Why you closed #1239
> This is not a good practice, you should learn how to fix conflicts instead of just deleting it and making a new PR.
> I think you should open that PR and work on that itself instead of creating this new one.

@SaranshDawra Please  have a look at this message, we will not merge this PR as this is not a good particle, open the last one and work on that itself.

**Comment by SaranshDawra** [comment]
@Kundan28 I am not able to reopen the previous PR as I forked the repository again and started from the beginning.

**Comment by Kundan28** [comment]
> @Kundan28 I am not able to reopen the previous PR as I forked the repository again and started from the beginning.

@alodahl Could we allow him to work on this PR ?

File: career/professionalism.md [hunk 1]
```diff
@@ -13,8 +13,8 @@ About 2 hours
 
 ### Prerequisites
 - [Expectations While at Techtonica](https://docs.google.com/document/d/1IWny-IXB-XDWW21l3JODwPuh1QOVBFQbii_DjvF0g00/edit?usp=sharing)
-- [Interpersonal Conflict Resolution](/conflict-resolution/conflict-resolution.md)
-- [Diversity, Inclusion & Implicit Bias](/diversity-inclusion-bias/inclusion.md)
+- [Interpersonal Conflict Resolution](conflict-resolution.md)
```
Comment (alodahl) [inline career/professionalism.md]: this link is broken.

**Comment by alodahl** [comment]
Yup, jut follow the command line links below, they'll give you directions.

**Comment by SaranshDawra** [comment]
Screenshot (100)

@alodahl The link is not showing here.

**Comment by alodahl** [comment]
I will merge soon. Go ahead and pick up a new issue @SaranshDawra 😊

**Comment by Kundan28** [comment]
Nice work @SaranshDawra

**Pull request merged**